### PR TITLE
fix changeLog style

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**
#### Which User story does this relate to (link)?
[-COG-1066](https://osi-cwds.atlassian.net/browse/COG-1066)

#### Brief feature description
-Style fix for ChangeLog Component

### CODE REVIEW

File: app/javascript/views/usersList/UsersList.scss [hunk 1]
```diff
@@ -35,7 +35,7 @@
     .-previous,
     .-next {
       flex: 1 1 40px;
-      margin-top: 12.5px;
+      margin-top: 5.5px;
```
Comment (kelfink) [inline app/javascript/views/usersList/UsersList.scss]: is this to fix the little box showing near the dropdown in safari?

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -17,7 +17,20 @@
 @import 'bootstrap-sprockets';
 @import 'bootstrap';
 @import 'react-wood-duck/dist/styles/application.scss';
-@import '@cwds/core/dist/styles.css';
+@import '@cwds/core/dist/styles';
+
+body {
```
Comment (kelfink) [inline app/assets/stylesheets/application.scss]: this looks very global.  affects all pages, doesn't it ?

Comment (Sushmareddy-OSI) [reply app/assets/stylesheets/application.scss]: It is overriding with the component library styles.

File: app/javascript/views/usersList/UsersList.scss (hunk 1 above)
Comment (Sushmareddy-OSI) [reply app/javascript/views/usersList/UsersList.scss]: No, with the new update of component library, the styles for pagination is breaking. This is for the previous and next buttons in pagination

Comment (kelfink) [reply app/javascript/views/usersList/UsersList.scss]: oh, of course "-next"

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (kelfink) [reply app/assets/stylesheets/application.scss]: Ok.    It does seem strange we need to do this.  I wonder if the component library is intent on changing our styles for our apps, or if it should not be really using styles on body.  @eckdanny-osi might weigh in here when available.

**Pull request merged**